packages/documentation/src/stories/get-started/icons.stories.mdx

```diff
@@ -0,0 +1,122 @@
+import { Meta, Source } from '@storybook/addon-docs';
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
+
+<Meta
+  title="Get Started/Icons"
+  parameters={{
+    previewTabs: {
+      canvas: {
+        hidden: true,
+      },
+    },
+    viewMode: 'docs',
+    badges: [BADGE.NEEDS_REVISION],
+  }}
+/>
+
+# @swisspost/design-system-icons
+
+[![npm version](https://badge.fury.io/js/@swisspost%2Fdesign-system-icons.svg)](https://badge.fury.io/js/@swisspost%2Fdesign-system-icons)
+
+There are currently two totally different approches to use our icons.<br/>
+
+- In the first approch, you can use our icons as an **svg-only** library.<br/>
+- In the second aproch, you can use our flexible, lightweight and easy-to-use **post-icon** component.
+
+Decide for yourself, which way fits best for you...
+
+More Informations about the icons can be found under the [Icons Search](/?path=/docs/foundations-icons--page) or the [Icons Component](/?path=/docs/components-post-icon--default) documentation.
+
+## Svg-only approch
```

Comment:
I'd promote the `post-icon` way as the default way of using icons. For most projects, the components will already be available because they are in the blueprint.